WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -156,6 +156,7 @@ - (void)updateFeedSourcesUserInitiated:(BOOL)wasUserInitiated completion:(nullab
 
 - (void)updateFeedSourcesWithDate:(nullable NSDate *)date userInitiated:(BOOL)wasUserInitiated completion:(nullable dispatch_block_t)completion {
     WMFAssertMainThread(@"updateFeedSources: must be called on the main thread");
+    [FeedFunnel.shared logFeedRefreshed];
```

Comment:
Does it matter here if the user refreshed it? I would have assumed it would only  be `if (wasUserInitiated)` so background updates aren't logged

Reply:
yeah, good point, maybe this should be in Explore's `refresh` and `scrollViewDidScroll` (Chelsy wanted to log feed update on scroll too)